pkg/v1/tkg/tkgpackageclient/package_update.go

```diff
@@ -117,6 +119,13 @@ func (p *pkgClient) updateValuesFile(o *tkgpackagedatamodel.PackageOptions, pkgI
 		}
 	}
 
+	pkgInstallToUpdate.Spec.Values = []kappipkg.PackageInstallValues{
+		{
+			SecretRef: &kappipkg.PackageInstallValuesSecretRef{
+				Name: fmt.Sprintf(tkgpackagedatamodel.SecretName, o.PkgInstallName, o.Namespace)},
```

Comment:
Line 107 and 125 are the same, can we create a util function to get the secret name from there? It will be easier to manage and change later.

Or just reuse `o.SecretName`.